From c70b2b402ac1ca3a923822ddd2128a09ce5de81a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Sveinar=20S=C3=B8pler?= <cybermax@dexter.no>
Date: Mon, 24 Jan 2022 16:02:35 +0100
Subject: [PATCH] nvcuda: Make nvcuda attempt to load libcuda.so.1

libcuda.so is not available when running under steam runtime
container. Adding this will make nvcuda attempt to load
libcuda.so.1 which will be available inside the steam runtime.
---
 dlls/nvcuda/nvcuda.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/dlls/nvcuda/nvcuda.c b/dlls/nvcuda/nvcuda.c
index 2cb3a0e2ecf..84da77941c6 100644
--- a/dlls/nvcuda/nvcuda.c
+++ b/dlls/nvcuda/nvcuda.c
@@ -424,7 +424,8 @@ static BOOL load_functions(void)
         "/usr/local/cuda/lib/libcuda.dylib",
         "/usr/local/cuda/lib/libcuda.6.0.dylib",
     #else
-        "libcuda.so"
+        "libcuda.so",
+        "libcuda.so.1"
     #endif
     };
     int i;
-- 
2.34.1

